### PR TITLE
unzip: unzip extract non-ascii file

### PR DESCRIPTION
**Pull request by ramwin**
----

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/unzip.md [hunk 1]
```diff
@@ -17,3 +17,7 @@
 - Extract the contents of the file(s) to stdout alongside the extracted file names:
 
 `unzip -c {{file.zip}}`
+
+- Extract zip file created by windows (you may use the command if there are non-ascii filenames in zip file):
```
Comment (agnivade) [inline pages/common/unzip.md]: Is this option available only in windows ? Because I don't see it in `man unzip` output.

Comment (ramwin) [reply pages/common/unzip.md]: Sorry, maybe I didn't describe it clearly. If you use windows to create a zip file which contains non-ascii character in filenames and unzip it in linux system, the file name will become unreadable. See [this](https://github.com/girder/girder/issues/985) or [this](https://github.com/rubyzip/rubyzip/wiki/Files-with-non-ascii-filenames) for more information. In this case, you should use `unzip -O gbk {{file}}`. This `-O` option is available on my `ubuntu desktop 18.04` and `ubuntu server 16.10`.

Comment (agnivade) [reply pages/common/unzip.md]: What does the -O option do ? What does does gbk stand for ? I could not find -O here - https://linux.die.net/man/1/unzip. I hope you are not talking about -o.

Comment (sbrl) [reply pages/common/unzip.md]: Hrm, yeah. It looks like the man page leaves something to be desired....

**Comment by ramwin** [comment]
@agnivade Thank you. I looks better and more clear now.

**Pull request merged**